composer.json

```diff
@@ -11,13 +11,18 @@
             "email": "chobieeee@php.net"
         }
     ],
-    "minimum-stability": "dev",
+    "config": {
+        "platform": {
+            "php": "5.3.29"
```

Comment:
I have all PHP versions installed locally. If you'd like I can send diff to you with correct `composer.lock` file for your `composer.json` (without that `config` setting)?